<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:24px;
}
#centralwidget
{
background:#f7abcd;
}
QFrame
{
background:rgba(0, 0, 0, 0.2);
border-radius:15px;
}
QPushButton
{
background:#e0aa9d;
border-radius:60px;
}
QToolButton
{
background:red;
border-radius:60px;
}
QLabel
{
color:white;
}
QPushButton
{
color:white;
border-radius:15px;
}
QPushButton:hover
{
color:#333;
border-radius:15px;
background:#9e40fc;
}
QLineEdit
{
background:transparent;
border:none;
color:#d9d8d9;
border-bottom: 1px solid #d9d8d9;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1050</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1050</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QFrame" name="left_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>250</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="book_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
border-radius:15px;
color:#333;
background:rgb(197, 255, 250);
}
</string>
     </property>
     <property name="text">
      <string>Book</string>
     </property>
    </widget>
    <widget class="QPushButton" name="person_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
border-radius:15px;
color:#333;
background:rgb(197, 255, 250);
}</string>
     </property>
     <property name="text">
      <string>Person</string>
     </property>
    </widget>
    <widget class="QPushButton" name="purchase_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
border-radius:15px;
color:#333;
background:rgb(197, 255, 250);
}</string>
     </property>
     <property name="text">
      <string>Purchase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_all_records">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>230</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Clear all records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_database">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>230</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Delete database</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="start_frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background:rgba(0, 0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="book_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>721</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
    </widget>
    <widget class="QTableWidget" name="person_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>721</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
    </widget>
    <widget class="QTableWidget" name="purchase_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>741</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>170</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>170</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>170</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="add_book_record">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Add new record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_book_record">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Delete the record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_delete_by_book">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Search/delete by book</string>
     </property>
    </widget>
    <widget class="QLabel" name="table2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>250</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(197, 255, 250);
color:black;
</string>
     </property>
     <property name="text">
      <string>            Tables</string>
     </property>
    </widget>
    <widget class="QLabel" name="table1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>90</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(197, 255, 250);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_person_record">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Add new record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_purchase_record">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Add new record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_person_record">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Delete the record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_delete_by_person">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Search/delete by person</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_delete_by_purchase">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Search/delete by purchase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_purchase_record">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Delete the record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_books_records">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Clear books records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_people_records">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Clear people records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_purchases_records">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>90</y>
       <width>340</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>70</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Clear purchases records</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
